Lib/concurrent/futures/thread.py

```diff
@@ -127,8 +127,8 @@ def __init__(self, max_workers=None, thread_name_prefix='',
         """
         if max_workers is None:
             # ThreadPoolExecutor is often used to:
-            # * CPU bound task which releases GIL
-            # * I/O bound task (which releases GIL, of course)
+            # * CPU-bound task which releases GIL
+            # * I/O-bound task (which releases GIL, of course)
```

Comment:
In both of these cases, `I/O bound` and `CPU bound` precede the word `task`, so they should be hyphenated.

["Use a hyphen in a compound modifier when the modifier comes before the word it’s modifying."](https://www.grammarly.com/blog/hyphen/)